src/components/Alert/Alert.test.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import renderWithRedux from '../../testUtils/renderWithRedux';
+import Alert from './Alert';
+
+describe('Alert', () => {
+  it('shows alert', () => {
+    const { getByTestId } = renderWithRedux(<Alert />, {
+      initialState: {
+        alert: {
+          showAlert: true,
+          type: 'error',
```

Comment:
Where is the field used?
I've checked Alert component itself and didn't find any place.

Reply:
the component alert is rendered in the app component https://prnt.sc/115o2km
and it is being triggered e.g when we could not fetch cart items https://prnt.sc/115o33w
updated the test in the next commit https://github.com/maxSedelnikov/react-shopping-app/pull/6/commits/c759e7206dd9006634852446e70ed64047e50a1d